helm/templates/conductor-ingress.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: conductor-ingress
+spec:
+  rules:
```

Comment:
Do we need an ingress if everything is supposed to go into kubernetes? I guess this due to CORS? TBH I'm not sure what the best practices are in this regard :/

Reply:
> Do we need an ingress if everything is supposed to go into kubernetes? 

Yes because the FE is on your browser so it needs to contact the API that needs to be exposed externally. Usually this is done through the ingress.  More info [here](https://medium.com/@seanlinsanity/how-to-expose-applications-running-in-kubernetes-cluster-to-public-access-65c2fa959a3b#:~:text=In%20Kubernetes%2C%20a%20NodePort%20service,is%20known%20as%20the%20NodePort.) and [here](https://konghq.com/blog/engineering/gateway-api-vs-ingress)